docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,329 @@
+------------------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **JSON API**
+     - 
+     -
+   * - HTTP endpoints under ``/v1/`` including status codes, authentication, query language and encoding.
+     - Stable
+     -
+   * - ``daml json-api`` CLI :ref:`for development <dev-tools>`. (as specified using ``daml json-api --help``)
+     - Stable
+     -
+   * - Stand-alone distribution for production use, including CLI specified in ``--help``.
+     - Stable
+     -
+   * - **Triggers**
+     -
+     -
+   * - :doc:`DAML API of individual Triggers </triggers/index>`
+     - Beta
+     -
+   * - :doc:`Development CLI to start individual triggers in dev environment </triggers/index>` (``daml trigger``)
+     - Beta
+     -
+   * - :doc:`/tools/trigger-service` (``daml trigger-service``)
+     - Alpha
+     -
+   * - **Extractor**
+     -
+     -
+   * - :doc:`/tools/extractor` (``daml extractor``)
+     - Labs
+     -
+
+.. _libraries:
+
+Libraries
+---------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **Scala Ledger API Bindings**
+     -
+     -
+   * - ``daml codegen scala`` :doc:`CLI and generated code </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - ``bindings-scala_2.12`` :doc:`library and its public API </app-dev/bindings-scala/index>`
+     - Stable
+     - 
+   * - **Java Ledger API Bindings**
+     - 
+     -
+   * - ``daml codegen java`` :doc:`CLI and generated code </app-dev/bindings-java/codegen>`
+     - Stable
+     -
+   * - ``bindings-java`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - ``bindings-rxjava`` :doc:`library and its public API </app-dev/bindings-java/index>`.
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.6-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.7-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-1.8-archive-java-proto``
+     - Stable
+     -
+   * - Maven artifact ``daml-lf-dev-archive-java-proto``
+     - Alpha
+     -
+   * - **Node.js Ledger API Bindings**
+     -
+     -
+   * - ``@digital-asset/binings-js`` :doc:`Node.js library </app-dev/bindings-js>`
+     - Stable
+     - 
+   * - **JavaScript Client Libraries**
+     -
+     -
+   * - ``daml codegen js`` :doc:`CLI and generated code </app-dev/bindings-ts/daml2js>`
+     - Stable
+     -
+   * - ``@daml/types`` :doc:`library and its public API </app-dev/bindings-ts/daml-types>`
+     - Stable
+     -
+   * - ``@daml/ledger`` :doc:`library and its public API </app-dev/bindings-ts/daml-ledger>`
+     - Stable
+     -
+   * - ``@daml/react`` :doc:`library and its public API </app-dev/bindings-ts/daml-react>`
+     - Stable
+     -
+   * - **DAML Libraries**
+     -
+     -
+   * - :doc:`The DAML Standard Library </daml/stdlib/index>`
```

Comment:
Technically split into daml-stdib and daml-prim, not sure if you want to mention this. We merge the docs but the difference can still be observed and is exposed to users to some degree.

Reply:
Thought about it and decided not to split it here. If you feel strongly the other way, let me know, either is fine with me.

Reply:
Let’s leave it combined. Otherwise, you also have to split up the separate stable LF packages and that just gets unnecessarily messy.